cmd/zpool/zpool_main.c

```diff
@@ -8705,7 +8711,7 @@ zpool_do_status(int argc, char **argv)
 		(void) fsleep(interval);
 	}
 
-	return (0);
+	return (cb.cb_problematic);
```

Comment:
Ouch, it's weird for me to give non-zero exit code here. If we really want to do it - it looks better for me to give same exit code on one or more problematic pools, and describe it in separate `EXIT STATUS` section with other variants.